mlrun/utils/model_monitoring.py

```diff
@@ -108,13 +110,41 @@ def set_project_model_monitoring_credentials(
     :param project: The name of the model monitoring project.
     """
     mlrun.get_run_db().create_project_secrets(
-        project=project or config.default_project,
+        project=project or mlrun.mlconf.default_project,
         provider=mlrun.api.schemas.SecretProviderName.kubernetes,
         secrets={"MODEL_MONITORING_ACCESS_KEY": access_key},
     )
 
 
-class EndpointType(IntEnum):
+class EndpointType(enum.IntEnum):
     NODE_EP = 1  # end point that is not a child of a router
     ROUTER = 2  # endpoint that is router
     LEAF_EP = 3  # end point that is a child of a router
+
+
+class TrackingPolicy(mlrun.model.ModelObj):
+    """
+    Modified model monitoring configurations. By using TrackingPolicy, the user can apply his model monitoring
+    requirements, such as setting the scheduling policy of the model monitoring batch job or changing the image of the
+    model monitoring stream.
+    """
+
+    def __init__(
+        self,
+        default_batch_intervals: str = "0 */1 * * *",
```

Comment:
Why you are not using schedule object?

Reply:
Right, changed to `ScheduleCronTrigger`. Note that it required few more changes in the code.